Source/EasyNetQ.Hosepipe.Tests/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net40" />
   <package id="NUnit" version="2.6.3" targetFramework="net40" />
-  <package id="RabbitMQ.Client" version="3.5.4" targetFramework="net40" />
+  <package id="RabbitMQ.Client" version="3.5.5" targetFramework="net4" />
```

Comment:
I thought so too, this what VS2015 with the latest nuget package manager generated though, should I change it back?